compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -285,3 +298,141 @@ internal class DelegateRootForTestListener : PlatformContext.RootForTestListener
         }
     }
 }
+
+/**
+ * Platform specific text input service.
+ *
+ * This is a non-deprecated version of [PlatformTextInputService], which is needed because we can't
+ * expose deprecated APIs in [PlatformContext].
+ */
+interface PlatformContextTextInputService {
```

Comment:
Why should it be a public stable API?

Reply:
Please move into a separate file

Reply:
> Why should it be a public stable API?

As the comment above it explains, we can't expose a deprecated API in `PlatformContext`.